api/apps/v1alpha1/nimservice_types.go

```diff
@@ -608,6 +615,7 @@ func (n *NIMService) GetServiceParams() *rendertypes.ServiceParams {
 
 	// Set service ports
 	params.Port = n.GetServicePort()
+	params.PortName = "open-ai-port"
```

Comment:
Currently, the metrics port for embedding/reranking if not the open-ai-port. It is a separate port.